docs/application-development-guide.md

```diff
@@ -3132,6 +3132,55 @@ const handle = await client.start(scheduledWorkflow, {
 
 TODO
 
+<Tabs
+defaultValue="go"
+groupId="site-lang"
+values={[{label: 'Go', value: 'go'},{label: 'Java', value: 'java'},{label: 'PHP', value: 'php'},{label: 'Typescript', value: 'typescript'},]}>
+
+<TabItem value="go">
+
+Content is not available
+
+</TabItem>
+<TabItem value="java">
+
+Content is not available
+
+</TabItem>
+<TabItem value="php">
+
+To create a local Activity, use `#[LocalActivityInterface]`.
+
+```php
+use Temporal\Activity\LocalActivityInterface;
+
+#[ActivityInterface]
+interface YourActivityInterface
+{
+    #[ActivityMethod("youractivity")]
+    public function youractivity(): string;
+}
+```
+
+Local activity requires `LocalActivityOptions`:
+
+```php
+$greetingActivity = Workflow::newActivityStub(
+    YourActivityInterface::class,
+    LocalActivityOptions::new()->withStartToCloseTimeout(\DateInterval::createFromDateString('30 seconds'))
+);
+```
+
+Local Activities have a limited set of options, including: `ScheduleToCloseTimeout`, `StartToCloseTimeout`, `RetryPolicy`.
```

Comment:
When you have a list of three or more items, put it in a bulleted list.

Reply:
Suggested change below